src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -35,8 +37,8 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
             _writer.WriteObjectStart(); // root
             _writer.Write("version", OutputFormatVersion);
 
```

Comment:
if you emit a $schema property, your emitted SARIF will automatically validate when opened in VS or VS Code.

"$schema": "[http://json.schemastore.org/sarif-1.0.0](http://json.schemastore.org/sarif-1.0.0)"